functions/venue.js

```diff
@@ -650,7 +650,7 @@ exports.updateVenue_v2 = functions.https.onCall(async (data, context) => {
     updated.radioStations = [data.radioStations];
   }
 
-  if (data.mapBackgroundImageUrl) {
+  if (typeof data.mapBackgroundImageUrl === "string") {
```

Comment:
What's the intent of this change here? I'm guessing because `""` is falsy, and that's what you said the 'default' is set to currently, so this wouldn't change it previously?

Reply:
Yes, that's correct.

Reply:
Makes sense.